rules.js

```diff
@@ -186,6 +186,22 @@ class sub_with_return extends rule(3) {
     }
 }
 
+class yoda_condition extends rule(3) {
+    constructor() {
+        super()
+        this._node = 'bop'
```

Comment:
can you give a little explanation for this? is this how we filter for expressions that are binary operations?

Reply:
Yes. This rule will only fire for binary operations, including conditional operations.